.buildkite/scripts/steps/integration_tests.sh

```diff
@@ -4,11 +4,21 @@ set -euxo pipefail
 source .buildkite/scripts/common.sh
 
 # PACKAGE
-DEV=true EXTERNAL=true SNAPSHOT=true PLATFORMS=linux/amd64,linux/arm64 PACKAGES=tar.gz mage package
+AGENT_PACKAGE_VERSION=8.10.0 \
```

Comment:
```suggestion
AGENT_PACKAGE_VERSION=8.10.0-SNAPSHOT \
```

This will fix the failing upgrade test at least.